### PR TITLE
Use f-strings and lint with black

### PR DESCRIPTION
**Pull request by lmmx**

### CODE REVIEW

**Comment by lmmx** [comment]
Submitted this in a hurry at the end of the night and didn’t consider running the tests — will take a look and resubmit tomorrow

I wonder why your tests for black linting is failing for the code I ran through black myself...

File: docs/source/conf.py [hunk 1]
```diff
@@ -12,14 +12,15 @@
 #
```
Comment (rlouf) [inline docs/source/conf.py:12]: You ran the docs' config through black! Better restore this one.

**Comment by rlouf** [comment]
Thank you for the f-strings, that's helpful!

I've had that happen on my machine and it was due to using a different version on black inside and outside my virtual environment. I haven't vendorized the version yet, and the CI is currently using the latest version (20.8b1). Maybe try this?

Tests were failing but it was my fault. You should rebase on master before re-running them.

Note that there a Makefile that reproduces the CI's steps for linting. Try `make ci` when you are in the root folder. If you're feeling fancy you can add a git hook that runs the command before `git push` :)

**Comment by lmmx** [comment]
Going to send a fresh PR as there are now commits in your repo I need to reflect, closing

**Pull request closed without merging**